src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -1328,7 +1327,7 @@ void InputProcessor::lowerRangeCheck(EnergyPlusData &state,
         Message2 += fmt::format(", range={{{}}}", LowerBoundString);
 
         {
-            auto const errorCheck(ErrorString[0]);
+            char const errorCheck = ErrorString[0];
```

Comment:
Scalars should not be `auto`. Also don't use constructor notation to initialize scalars which don't have constructors. It's not cool, it's annoying. And it messes up my `grep` scripts.